### PR TITLE
chore: refactor metaverse service

### PR DESCRIPTION
**Pull request by pedrotambo**
There was a component called `deployer` implementing this interface:
```
interface MetaverseContentService {
  deployEntity
  isContentAvailable
  getContent
  getDeployments
}
```
As we may see, all of these methods are related to deployments but only `deployEntity` is something that you would expect from a `"deployer"`. So we can split the operations into two groups:

1. Read operations:   `isContentAvailable`, `getContent` , `getDeployments`, `getDeploymentsForActiveEntities`
2. Write operation: `deployEntity`.

In addition, we can see in the code that the read operations don't use the logic in the `deployEntity` logic. For the sake of simplicity and separation of responsibility, the logic between the two groups was separated in two different files.

To summarize the changes proposed in this PR are:

- The read operations logic was moved to the already existing file `src/logic/deployments.ts`.
- The write operation logic was moved to a new component called `Deployer` in `src/ports/deployer.ts`.
- All types associated with both operations that were defined in multiple files all across the code were condensed and moved to a new file `src/deployment-types.ts`.

### CODE REVIEW

**Comment by menduz** [comment]
THANK YOU SO MUCH FOR REMOVING THAT NAME

File: content/src/logic/deployments.ts [hunk 1]
```diff
@@ -138,3 +149,157 @@ export async function calculateOverwrites(
     overwrittenBy
   }
 }
+
+export async function getContent(
```
[the remark this answers is not in the document]
Comment (pedrotambo) [reply content/src/logic/deployments.ts:153]: Totally. Just wanted to leave those kinds of refactors to another PR. BUT, given the simplicity of `getContent` and `isContentAvailable` will do it here.

**Review by hugoArregui — APPROVED** [review]
